src/models/Roles/AdminRole.tsx

```diff
@@ -1,14 +1,15 @@
 export enum AdminRole {
-  Admin = 0, //Admin
-  GoverningBodyHead = 1, //Голова керівного органу
-  OkrugaHead = 2, //Голова округи
-  OkrugaHeadDeputy = 3, //Заступник голови округи
-  CityHead = 4, //Голова станиці
-  CityHeadDeputy = 5, //Заступник голови станиці
-  KurinHead = 6, //Голова куреня
-  KurinHeadDeputy = 7, //Заступник голови куреня
+  Admin = 0, // Admin
+  GoverningBodyAdmin = 1, // Крайовий адмін
+  GoverningBodyHead = 2, // Голова керівного органу
+  OkrugaHead = 3, // Голова округи
+  OkrugaHeadDeputy = 4, // Заступник голови округи
+  CityHead = 5, // Голова станиці
+  CityHeadDeputy = 6, // Заступник голови станиці
+  KurinHead = 7, // Голова куреня
+  KurinHeadDeputy = 8, // Заступник голови куреня
```

Comment:
It is not needed to enumerate them yourself

Reply:
Thank, I understand)